mw-webapp/src/component/select/Select.tsx

```diff
@@ -142,6 +147,11 @@ export interface SelectProps<T> {
    * Custom class name
    */
   className?: string;
+
+    /**
+     * Data attributes for cypress testing
+     */
+  dataCy?: string;
```

Comment:
1. PLease check spaces
2/ We have already cy prop for SelectProps interface. Is it really needed to use addinition dataCy prop?

Reply:
If I delete it I get an error, I don’t know how to use the existing one.